phpBB/phpbb/db/migration/data/v33x/extend_bbcode_tooltip.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v33x;
+
+class extend_bbcode_tooltip extends \phpbb\db\migration\migration
+{
+	public function effectively_installed()
+	{
+		return $this->db_tools->sql_table_exists($this->table_prefix . 'bbcodes');
```

Comment:
If I see this correctly, this migration will not be applied when the bbcodes table already exists. This should however be the case for all 3.x boards.